app/src/main/java/com/yacgroup/yacguide/RegionManagerActivity.kt

```diff
@@ -159,4 +177,21 @@ class RegionManagerActivity : BaseNavigationActivity() {
             _updateHandler.finish()
         }
     }
+
+    private fun _selectDefaultRegion(regionId: Int) {
+        val key = getString(R.string.default_region)
```

Comment:
Why is the German word `Standardgebiet` used internal as settings key? Because this string is defined in a file which is used to translate the app into other languages, this key changes for each language. It makes no sense and even can lead to the fact that the settings get lost, if the language is changed. I know, at the moment this cannot happen because the app is only available in German but in general it should be avoided. Why not just defining it in the `setup.xml`?